charms/katib-controller/tests/integration/test_charm.py

```diff
@@ -24,20 +24,20 @@
+    "katib-config.yaml": "---\napiVersion: config.kubeflow.org/v1beta1\nkind: KatibConfig\ninit:\n  controller:\n    webhookPort: 443\n    trialResources:\n      - Job.v1.batch\n      - TFJob.v1.kubeflow.org\n      - PyTorchJob.v1.kubeflow.org\n      - MPIJob.v1.kubeflow.org\n      - XGBoostJob.v1.kubeflow.org\n      - MXJob.v1.kubeflow.org\nruntime:\n  metricsCollectors:\n    - kind: StdOut\n      image: docker.io/kubeflowkatib/file-metrics-collector:v0.17.0-rc.1\n    - kind: File\n      image: docker.io/kubeflowkatib/file-metrics-collector:v0.17.0-rc.1\n    - kind: TensorFlowEvent\n      image: docker.io/kubeflowkatib/tfevent-metrics-collector:v0.17.0-rc.1\n      resources:\n        limits:\n          memory: 1Gi\n  suggestions:\n    - algorithmName: random\n      image: docker.io/kubeflowkatib/suggestion-hyperopt:v0.17.0-rc.1\n    - algorithmName: tpe\n      image: docker.io/kubeflowkatib/suggestion-hyperopt:v0.17.0-rc.1\n    - algorithmName: grid\n      image: docker.io/kubeflowkatib/suggestion-optuna:v0.17.0-rc.1\n    - algorithmName: hyperband\n      image: docker.io/kubeflowkatib/suggestion-hyperband:v0.17.0-rc.1\n    - algorithmName: bayesianoptimization\n      image: docker.io/kubeflowkatib/suggestion-skopt:v0.17.0-rc.1\n    - algorithmName: cmaes\n      image: docker.io/kubeflowkatib/suggestion-goptuna:v0.17.0-rc.1\n    - algorithmName: sobol\n      image: docker.io/kubeflowkatib/suggestion-goptuna:v0.17.0-rc.1\n    - algorithmName: multivariate-tpe\n      image: docker.io/kubeflowkatib/suggestion-optuna:v0.17.0-rc.1\n    - algorithmName: enas\n      image: docker.io/kubeflowkatib/suggestion-enas:v0.17.0-rc.1\n      resources:\n        limits:\n          memory: 400Mi\n    - algorithmName: darts\n      image: docker.io/kubeflowkatib/suggestion-darts:v0.17.0-rc.1\n    - algorithmName: pbt\n      image: docker.io/kubeflowkatib/suggestion-pbt:v0.17.0-rc.1\n      persistentVolumeClaimSpec:\n        accessModes:\n          - ReadWriteMany\n        resources:\n          requests:\n            storage: 5Gi\n  earlyStoppings:\n    - algorithmName: medianstop\n      image: docker.io/kubeflowkatib/earlystopping-medianstop:v0.17.0-rc.1",  # noqa: E501
```

Comment:
I bet this is pretty annoying to update, should we use some variable KATIB_VERSION="v0.17.0-rc.1" and just use it everywhere?

Same in the ones beneath.

Reply:
good suggestion, I added commit
note that I had to use the old string formatting `%` because with the new f-strings I was having trouble with the curly braces that already exist in the string

Reply:
lovely